### ISSUE TITLE
Add 'Schedule' section

### ISSUE THREAD

**Issue by oscard0m** [emberfest/emberfest.github.io#327]
# Speakers[^1]

| talk | speaker | status |
|------|---------|--------|
| **Universal Design: A new process for better apps** | `Melanie Sumner` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/347 |
| **Bootstrap your dashboard the right way** | `Alexis Falaise` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Move Fast and (Don't) Break Things** | `Gonçalo Morais` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Automated Visual Testing in Ember** | `Ignace Maes` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **A framework for ambitious companies** | `Aymeric Augustin (Invited Speaker)` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Two Ember Apps Become One** | `by Mo Fiebiger (she/her)` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Performance culture. New opportunities with Embroider** | `Tomek Nieżurawski` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Building a phygital platform with Ember.js** | `Dries de Lange` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/341 |
| **Bottled Ember - Batteries Included Web Framework, tiny tiny living space** | `Chis Manson` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/349|
| **{{should-avoid this.renderModifiers butHow=true}}** | `Aaron Chambers` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Test as if you’re in someone else’s shoes** | `Floris List` | <kbd>✅ done</kbd> via https://github.com/emberfest/emberfest.github.io/pull/297 |
| **Starbeam, and what it means for Ember by Yehuda Katz** | `Yehuda Katz` | <kbd>⚠️ Missing talk description</kbd> |
| **Routing in Polaris** | `Edward Faulkner` | <kbd>⚠️ Missing talk description</kbd> |
| **Panel Day 1: Data in Ember** | ??? | <kbd>⚠️ Missing panel description</kbd> <kbd>⚠️ Missing panelists</kbd> |
| **Panel Day 2: The Future of Ember** | ??? | <kbd>⚠️ Missing panel description</kbd> <kbd>⚠️ Missing panelists</kbd> |
| **Lightning talks** | 🔮 announced during the conf | ✅ announced during the conf |

# [Preview Link](https://deploy-preview-326--silly-leavitt-2e0668.netlify.app/)

# Schedule
A tentative schedule [in a Google doc, here](https://docs.google.com/spreadsheets/d/1y1OfaWkPSVGPUIwRnXgSsYJprjqmaajeh2YtiA9clgg/edit?usp=sharing).

# Subtasks
- [x] https://github.com/emberfest/emberfest.github.io/pull/297
- [x] https://github.com/emberfest/emberfest.github.io/pull/333
- [x] https://github.com/emberfest/emberfest.github.io/pull/332
- [x] https://github.com/emberfest/emberfest.github.io/pull/325
- [x] https://github.com/emberfest/emberfest.github.io/issues/334
- [x] https://github.com/emberfest/emberfest.github.io/issues/329
- [x] #339
- [x] #344
- [x] #346
- [x] #340
- [x] #345
- [ ] #337
- [ ] #338
- [x] ~Add missing information for Lightning Talks~ announced during the conf
- [x] #343

[^1]: This issue contains some relevant conversations we had in https://github.com/emberfest/emberfest.github.io/pull/297

**Comment by balinterdi** [comment]
I asked Dries for the missing picture hopefully he'll get back shortly. If not, he'll be the mustachoed man :D 

For Chris Manson, [could you use his photo from last year](https://emberfest.eu/2021/schedule/). His bio can stay the same, too:

> Developer at @simplabs, creator of Empress and member of the Ember Core Learning Team 

For Yehuda, you have the photo already, and I'm going to come up with a description soon.

For Ed, [you have everything but the talk title and description from last year](https://emberfest.eu/2021/schedule/), too. @marcoow Do you know what he's going to talk about this year? If not, I'll ask him.

I think that wraps it up, thank you!

**Comment by oscard0m** [comment]
Checking the schedule I added some extra points I miss.

I would recommend opening the preview link so you visually review all the things missing or need to be changed.

**Comment by balinterdi** [comment]
## Building a phygital platform with Ember.js by Dries de Lange

#### Description

Have you ever been to a museum where you used an app or touchscreen to deep dive into a subject? At Ocular, we build phygital applications for museums and showrooms, all with Ember.js. This talk will cover how we started with Ember.js, and how we went from physical installations to virtual showrooms.

#### Bio

Front-end engineer, working with Ember.js for five years. At Ocular, I develop web-based applications for museums, exhibitions and company showrooms for various audiences. My goal is to build apps that are innovative, creative and helpful to people.

![dries-de-lange](https://user-images.githubusercontent.com/5022/188069381-89f28934-94c0-4e79-966e-26a21b4cf819.png)

**Comment by balinterdi** [comment]
I just uploaded Dries's talk details.

I checked again and realized we do know what the panels will be about:

- Day 1 panel: "Data in Ember"
- Day 2 panel: "The Future of Ember"

We won't know what the Lightning Talks will be about in advance as it's only decided there, at the conference (people need to sign up there), so we should just have "Lightning Talks" in the schedule.

I'll leave a comment with Yehuda's talk details.

**Comment by balinterdi** [comment]
#### Starbeam, and what it means for Ember by Yehuda Katz

#### Description

(Missing, "I don't think we can get him to fill that out within the next 6 months…" said Marco :) )

#### Bio

Yehuda Katz is passionate about all things Open Source.

He is one of the creators of Ember.js, a member of the Rust Core Team, and a retired member of the Ruby on Rails and jQuery Core Teams. He is a member of ECMAScript's TC39 standards committee, and a former member of the W3C's TAG (Technical Architecture Group).

His 9-to-5 home is at the startup he founded, [Tilde Inc.](http://www.tilde.io/). There he works on [Skylight](https://www.skylight.io/), the smart profiler for Rails, and does Ember.js consulting.

**Comment by balinterdi** [comment]
@oscard0m @inesilva I also updated the schedule the Google Doc to reflect what I wrote here, above.

The goal would be to publish the schedule today. Are you guys working today on this, and could you do the few final updates so that we can do it?

Thank you in advance.

**Comment by oscard0m** [comment]
We are checking it, yes.

**Comment by oscard0m** [comment]
I created issues to work on them. I think we will be missing some info for:
- Chris Manson's talk description
- Ed's talk title and description

**Comment by balinterdi** [comment]
Chris Manson's talk description:

> All Ember addons come with the concept of a “dummy app”. This has been super useful to test our addons and has even been part of the reason why we have been able to support so many different versions of Ember from a single addon using ember-try scenarios. This talk gives a bit of detail on how things work with “classic” V1 addons and dummy apps and talks a little bit about why this can’t work for new Embroider V2 addons.

> I will then go into a detail on a particular setup that could bring the simplicity and magic of dummy apps to the standards focused Embroider world. This should make it super easy for developers to tests their apps in all the versions of Ember that they want to.

Ed's talk title:

> Routing in Polaris

It doesn't yet have a description, we might add it later

Melanie's talk title and description:

> "Universal Design: A new process for better apps"

> In this talk, you will learn about Universal Design: what it is and what it isn't; the principles behind Universal Design; how it's already being used in other industries; and how you can apply some of these principles while building your apps for improved outcomes for all users.

For the Lightning talks, could you remove TBA from both places? Attendees are familiar with the format so they are aware we'll all only know the exact talks shortly before the actual talks.

**Comment by oscard0m** [comment]
Changes applied.

I guess we need the Panels details and we are ready to land

**Comment by balinterdi** [comment]
@oscard0m I think we're almost there.

1. The panels:

- Data in Ember on Day 1
- The Future of Ember on Day 2

We'll add the panelists and details later but could you remove the content from last year in the meantime?

2. Also, could you remove the TBAs from talk details we don't have yet?

3. I noticed that all the speakers have rectangle-shaped photos, except Chris who still has a round one:

<img width="1632" alt="Screenshot 2022-09-02 at 14 10 13" src="https://user-images.githubusercontent.com/5022/188140052-1d8c3963-24d4-4c2d-b9ba-0efa446f068a.png">

Could you make it a rectangle, too? ([Here's his photo from Twitter](https://pbs.twimg.com/profile_images/886878264110850050/5oKQrMJi_400x400.jpg))

4. Last, but not least, the conference is on Sept 22-23 while the schedule says Sep 30 for Day 1 and Oct 1 for Day 2! I think this was copied from last year but I'm positive this would confuse people 😬

**Comment by oscard0m** [comment]
@inesilva can you takes this, please? :)

**Comment by inesilva** [comment]
@balinterdi all those things are done, let me know if we need to change or add something else

**Comment by balinterdi** [comment]
@inesilva @oscard0m Hmm, ignore my previous comment, when I clicked the Preview link at the top of this issue, I do see the fixes.

1. The dates are still wrong on the full schedule (`/schedule` page):

<img width="1901" alt="Screenshot 2022-09-02 at 19 08 17" src="https://user-images.githubusercontent.com/5022/188204292-7490fbb6-6fe7-439b-a362-77b340146716.png">

2. Is there a way I could check out the page *locally* before going live (which means merging #326, if I'm not mistaken). Shouldn't the way I did it (see my previous comment) work? I would publish the schedule this weekend and you shouldn't need to be here for this to happen. You guys should be on the beach, in a museum, or playing board games with friends. Anything but merging PRs and adding CSS classes to elements.
